### PR TITLE
Extra features --added

### PR DESCRIPTION
**Pull request by itsmebipul00**
Remember me --added
Forgot Password --added
Product View Page --added

### CODE REVIEW

File: Product.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              <div class="card-footer">
+
+                <p class="card__price fs-500"><strong>PRICE:  </strong><span class="line-through text-dark-70  original-price"> $345</span>$50</p>
+                <button class="btn btn-icon fs-300">
+                    <span class="margin-r-auto">ADD TO CART</span> 
+                    <i class="fas fa-shopping-cart fa-lg"></i>
+                </button>
+                
+                
+
```
Comment (Muzammil-khan710) [inline Product.html:64]: extra space

**Review by Muzammil-khan710 — APPROVED** [review]
Look's good to me. Good to merge

**Review by purvasharmadev — APPROVED** [review]

File: Products.html [hunk 2]
```diff
@@ -144,9 +159,9 @@ <h3 class="card__title">Marshmello</h3>
                     
                         <i class="far fa-heart fa-2x margin-left-auto"></i>
                     </div>  
-                </div>
-              </div>
-              <div class="card stacked">
+                </a>
+
+              <a class="card d-grid grid-stacked" href="./Product.html">
                 <img src="images/candy-1.jpg" alt="" class="card__img">
```
Comment (zoyatanweer) [inline Products.html:165]: write alt for the image

Comment (zoyatanweer) [inline Products.html:165]: write alt tag for image

File: Products.html [hunk 3]
```diff
@@ -158,9 +173,9 @@ <h3 class="card__title">Marshmello</h3>
...
```
Comment (zoyatanweer) [inline Products.html:179]: write alt for image

File: Products.html [hunk 4]
```diff
@@ -172,9 +187,9 @@ <h3 class="card__title">Marshmello</h3>
                     
                         <i class="far fa-heart fa-2x margin-left-auto"></i>
                     </div>  
-                </div>
-              </div>
-              <div class="card stacked">
+                </a>
+   
+              <a class="card d-grid grid-stacked" href="./Product.html">
                 <img src="images/candy-1.jpg" alt="" class="card__img">
```
Comment (zoyatanweer) [inline Products.html:193]: write alt

File: Products.html [hunk 5]
```diff
@@ -243,8 +258,8 @@ <h3 class="card__title">Marshmello</h3>
                         <i class="far fa-heart fa-2x margin-left-auto"></i>
                     
                 </div>
-              </div>
-              <div class="card stacked">
+              </a>
+              <a class="card d-grid grid-stacked" href="./Product.html">
                 <img src="images/candy-1.jpg" alt="" class="card__img">
```
Comment (zoyatanweer) [inline Products.html:263]: write alt for img

File: Products.html [hunk 6]
```diff
@@ -229,8 +244,8 @@ <h3 class="card__title">Marshmello</h3>
...
```
Comment (zoyatanweer) [inline Products.html:249]: write alt

File: Products.html [hunk 7]
```diff
@@ -215,8 +230,8 @@ <h3 class="card__title">Marshmello</h3>
                         <i class="far fa-heart fa-2x margin-left-auto"></i>
                 
                 </div>
-              </div>
-              <div class="card stacked">
+              </a>
+              <a class="card d-grid grid-stacked" href="./Product.html">
                 <img src="images/candy-1.jpg" alt="" class="card__img">
```
Comment (zoyatanweer) [inline Products.html:235]: write alt

File: Products.html [hunk 8]
```diff
@@ -201,8 +216,8 @@ <h3 class="card__title">Marshmello</h3>
                         <i class="far fa-heart fa-2x margin-left-auto"></i>
               
                 </div>
-              </div>
-              <div class="card stacked">
+              </a>
+              <a class="card d-grid grid-stacked" href="./Product.html">
                 <img src="images/candy-1.jpg" alt="" class="card__img">
```
Comment (zoyatanweer) [inline Products.html:221]: write alt

File: Products.html [hunk 9]
```diff
@@ -187,8 +202,8 @@ <h3 class="card__title">Marshmello</h3>
                         <i class="far fa-heart fa-2x margin-left-auto"></i>
                 
                 </div>
-              </div>
-              <div class="card stacked">
+              </a>
+              <a class="card d-grid grid-stacked" href="./Product.html">
                 <img src="images/candy-1.jpg" alt="" class="card__img">
```
Comment (zoyatanweer) [inline Products.html:207]: here also

**Review by zoyatanweer — APPROVED** [review]
code is very clean. Good to merge with suggested changes!

**Pull request merged**